src/Http/RequestUtil.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Http;
+
+use Flarum\User\User;
+use Psr\Http\Message\ServerRequestInterface as Request;
+
+class RequestUtil
+{
+    public static function getActor(Request $request): User
```

Comment:
I have a branch lying around somewhere that basically does the same thing, but more along the lines of `Actor::fromRequest()`. I liked that a lot. Maybe you like the idea, too? 😄

Reply:
Just saw more discussion in #2703 and related tickets. Have fun. 😀

Reply:
Actor is sensible too, but I prefer `RequestUtil` because:

- While getting it is clean, setting it results in clunky naming, as with the user method.
- If we need to encapsulate more stuff, we don't need to add more util classes / force people to import as much stuff. Perhaps separate classes is slightly better with respect to single responsibility, but I think encapsulating request data is still fairly granular. Better than adding it onto `User` anyhow :see_no_evil: